iot/api/manager/pom.xml

```diff
@@ -0,0 +1,82 @@
+<!--
+  Copyright 2017 Google Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.example.cloud</groupId>
+  <artifactId>cloudiot-manager-demo</artifactId>
+  <packaging>jar</packaging>
+  <version>1.0</version>
+  <name>cloudiot-manager-demo</name>
+  <url>http://maven.apache.org</url>
+
+  <!-- Start local repo -->
+  <repositories>
```

Comment:
I had intended to bundle the local repo in the sample, I'll add the jar and will add an instruction.